README.md

```diff
@@ -139,4 +148,10 @@ You should not be working on these (or even thinking about them) until you have
 - Create a CLI to interact with your hotel system
 
 ## What we're looking for
-You can find what instructors will be looking for in the [feedback](feedback.md) markdown document. 
+You can find what instructors will be looking for in the [feedback](feedback.md) markdown document.
+
+## Questions
+
+1. Does it make more sense to return an empty array or nil if there are no available rooms for a given date range?
+3. How best to organize/remember what functions I made and what I tested where?
```

Comment:
There are a lot of weird ways to approach this problem: one of them might be solved by different IDEs (alternatives for Atom) that could list out each method in each file, or some gems that could output the methods in each file, etc.

The above ideas are "weird" because Ruby is an interpreted language... There may be different ways to answer your question that I'll think about later. Feel free to poke other instructors to see what they think too.